library/Zend/Loader.php

```diff
@@ -130,10 +130,13 @@ public static function loadFile($filename, $dirs = null, $once = false)
         /**
          * Try finding for the plain filename in the include_path.
          */
-        if ($once) {
-            include_once $filename;
-        } else {
-            include $filename;
+        $absPath = stream_resolve_include_path($filename);
```

Comment:
Should this block be replaced with a call to the _includeFile static method that you modified below?

Reply:
Technically we could, but that method is marked as deprecated. We could remove that flag, but it was probably there to dissuade the direct usage of it.